frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -864,8 +885,12 @@ function DataFrame({
               // the same row after sorting, hover that would require us to map the selection
               // to the new index of the selected row which adds complexity.
               clearSelection()
+            } else {
+              // Cell selection are kept on the old position,
+              // which can be confusing. So we clear all cell selections before sorting.
+              clearSelection(true, true)
             }
-            sortColumn(colIndex)
+            sortColumn(colIndex, "auto")
```

Comment:
`"auto"` does not seem to be allowed based on the types (only `"asc"` & `"desc"`) 🤔

Reply:
This `sortColumn` is not the same as the one from the column menu. This one does support auto, while the callback from the column menu can only support asc & desc. I renamed the one from the column menu to `onSortColumn`